### PR TITLE
Add type tree

### PR DESCRIPTION
**Pull request by danielkingai2**
Add the type tree to the umls class, and a get parent function to the type tree

### CODE REVIEW

File: scispacy/umls_semantic_type_tree.py [hunk 1]
```diff
@@ -46,6 +46,20 @@ def get_children(self, node: SemanticTypeNode) -> List[SemanticTypeNode]:
             children.append(child)
             children.extend(self.get_children(child))
         return children
+    
+    def get_parent(self, node: SemanticTypeNode) -> Optional[SemanticTypeNode]:
```
Comment (DeNeutoy) [inline scispacy/umls_semantic_type_tree.py]: Might be nice to define `parent(0) = 0` so the return type is not optional, and likely this is what you want at the end of the day?

Comment (danielkingai2) [reply scispacy/umls_semantic_type_tree.py]: the test fixture type tree starts at level 1 :P

Comment (danielkingai2) [reply scispacy/umls_semantic_type_tree.py]: wasn't sure if there was a "right" answer here so just went with the catch all return None if no parents

Comment (DeNeutoy) [reply scispacy/umls_semantic_type_tree.py]: sure, if it's annoying to change the fixture no problem, lgtm

**Pull request merged**